public/index.html

```diff
@@ -7,6 +7,12 @@
     <meta name="theme-color" content="#000000" />
     <meta name="apple-mobile-web-app-capable" content="yes">
     <link rel="apple-touch-icon" href="assets/icon.png">
+    <!-- iPhone Xs Max (1242px x 2688px) --> 
```

Comment:
No need to add these comments. This information is already explicit on the `<link>` tag declarations.

Reply:
done